examples/rosettacode/rock_paper_scissor.nit

```diff
@@ -0,0 +1,50 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+#
+# Task: Programming_Task/Rock-paper-scissors
+# SEE: <http://rosettacode.org/wiki/Rock-paper-scissors>
+#
+# author: Istvan szalai 22/05/2015
+
+#return a value to dertermine who win the match
+fun win_or_lose(player, computer: Int): Int do
+	if player == computer then return 0
+	if player > computer then return 1
+	return 2
+end
+
+#return a string with about the match winner or looser
+fun result_of_the_match_message(number: Int): String do
+	if number == 0 then return "It's a tie !"
+	if number == 1 then return "You win"
+	return "You lose"
+end
```

Comment:
You should unify the last 2 methods. `win_or_lose` can directly print the result as a string. There would be less magic numbers!